### ISSUE TITLE
Presence of a border when running cli apps in termux

### ISSUE THREAD

**Issue by sudipta-nandan** [termux/termux-app#2177]
![Screenshot_20210719_004303_com.termux.jpg](https://user-images.githubusercontent.com/25419259/126079504-4355ef54-1e48-4a56-a631-bae7c6df5273.jpg)

I have using termux for almost an year... But the problem is this while using vim or emacs there is a border around the cli app. I'm using lucario theme for termux and material theme foe emacs. Is this normal to happen?

**Comment by xeffyr** [comment]
Yes, there is a small padding around terminal view. See this pull request as well as mentioned issue in description: https://github.com/termux/termux-app/pull/792

However it doesn't seem to be normal in your case. On the right side it should be same as on left.
